src/components/Feedback/Progress/FileProgress.tsx

```diff
@@ -35,16 +35,16 @@ const StyledButton = styled(Button)`
 `;
 
 interface TopProgressProps {
-  paused?: boolean;
-  error?: boolean;
+  $paused?: boolean;
```

Comment:
Is the $paused prop used? Cant see a change on adding the $ anywhere else in this file

Reply:
Good catch 🎣  I'll remove it